transforms/helpers/options.ts

```diff
@@ -1,49 +1,98 @@
+import { inspect } from 'node:util';
 import type { ZodError } from 'zod';
```

Comment:
TIL zod. What do you think about [ts-to-zod](https://www.npmjs.com/package/ts-to-zod)?

Reply:
I haven't tried ts-to-zod, but zod is already based on TS APIs and terminology, so it's super intuitive as-is.